dev/doc/README.md

```diff
@@ -6,28 +6,37 @@ See the first section of [`INSTALL`](../../INSTALL).  Developers are
 recommended to use a recent OCaml version, which they can get through
 opam or Nix, in particular.
 
-## Building `coqtop`
-The general workflow is to first setup a development environment with
-the correct `configure` settings, then hacking on Coq, make-ing, and testing.
+## Building `coqtop` / `coqc` binaries
 
+The recommended development workflow is to rely on Dune.
```

Comment:
```suggestion
We recommend you use Dune to build Coq rather than make.
```

Reply:
I use make to drive my dune targets every day, so the update is confusing; I'd write "we recommend to use th targets in Makefile.dune"